bin/feature-search.js

```diff
@@ -0,0 +1,153 @@
+#! /usr/env/bin node
+
+const fs = require( 'fs' );
+const chalk = require( 'chalk' );
+
+async function getConfigFeatures( config ) {
+	const file = `${ __dirname }/../config/${ config }.json`;
+	const json = await fs.readFileSync( file, 'utf8' );
+	const data = JSON.parse( json );
+
+	return data.features;
+}
+
+function calcLongestConfig( result ) {
+	let longestConfig = '';
+	for ( const item of result ) {
+		if ( item.config.length > longestConfig.length ) {
+			longestConfig = item.config;
+		}
+	}
+
+	return longestConfig.length;
+}
+
+function getPadding( count ) {
+	let padding = '';
+	for ( let i = 0; i < count; i++ ) {
+		padding += ' ';
+	}
+
+	return padding;
+}
+
+function sortResult( result ) {
+	result.sort( ( a, b ) => {
+		if ( a.config < b.config ) {
+			return -1;
+		}
+
+		if ( a.config > b ) {
```

Comment:
Should this not be comparing `b.config`?
```suggestion
		if ( a.config > b.config ) {
```